packages/rancher-prom-to-teams/generated-changes/patch/templates/NOTES.txt.patch

```diff
@@ -0,0 +1,8 @@
+--- charts-original/templates/NOTES.txt
++++ charts/templates/NOTES.txt
+@@ -1,2 +1,5 @@
+ Prom2Teams has been installed. Check its status by running:
+   kubectl --namespace {{ .Release.Namespace }} get pods -l "app.kubernetes.io/instance={{ .Release.Name }}"
++You can configure the Alertmanager to send alerts to the url:
++  http://{{- .Release.Name  -}}.{{- .Release.Namespace -}}.svc.cluster.local:8089/v2/<Connector1>
++Note: the <Connector1> is used to determine the receiver by Prom2Teams
```

Comment:
Let's drop this since it's never going to be installed independently of rancher-alerting-driver, so users will never see this.